velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -374,110 +358,85 @@ void AggregationFuzzer::go() {
       std::vector<TypePtr> argTypes = signature.args;
       std::vector<std::string> argNames = makeNames(argTypes.size());
 
-      // 10% of times test window operator.
+      const bool sortedInputs = FLAGS_enable_sorted_aggregations &&
+          canSortInputs(signature) && vectorFuzzer_.coinToss(0.2);
+
```

Comment:
Why was the removal of 10% window fuzzer replaced by 0.2 coin tosses ?

Reply:
Hi @natashasehgal, github doesn't show the code diff very well. You can see in [D69886065](https://www.internalfb.com/diff/D69886065) that I actually remove the entire if-body of the 10%-window-fuzzer testing.